clippy_lints/src/tabs_in_doc_comments.rs

```diff
@@ -137,7 +136,7 @@ fn get_chunks_of_tabs(the_str: &str) -> Vec<(u32, u32)> {
     if is_active {
         spans.push((
             current_start,
-            u32::try_from(the_str.chars().count()).expect(line_length_way_to_long),
+            u32::try_from(char_indices.last().unwrap().0 + 1).expect(line_length_way_to_long),
```

Comment:
Isn't this just

```suggestion
            u32::try_from(char_indices.len()).expect(line_length_way_to_long),
```

?

Reply:
I don't think so. [char_indicies](https://doc.rust-lang.org/std/primitive.str.html#method.char_indices) is an iterator over tuples with `(byte position, char)` and using the length of the iterator would produce the same result as the previous code. The suggestion would only work if all characters are 1-byte characters.

Reply:
Ah can you add a comment to that line than, that the first part is the byte position and not the character index? Otherwise r=me